efi/secureboot_policy.go

```diff
@@ -159,7 +159,7 @@ func (s *shimImageHandle) hasSbatSection() bool {
 	return s.openSection(".sbat") != nil
 }
 
-var shimIdentVersionRE = regexp.MustCompile(`^\$Version:[[:blank:]]*([[:digit:]]+)\.([[:digit:]]+)[[:blank:]]*\$$`)
+var shimIdentVersionRE = regexp.MustCompile(`^\$Version:[[:blank:]]*([[:digit:]]+)\.?([[:digit:]]+)?[[:blank:]]*\$$`)
```

Comment:
maybe the optional part can use a single (?:\\. ...)?

Reply:
It looks like that would convert the whole of the optional part in non-capturing.

Reply:
I think capturing groups are allowed inside non-capturing ones? unless my memory is off